.travis.yml

```diff
@@ -1,6 +1,7 @@
 language: python
 python:
 - 2.7
+- 3.4
```

Comment:
I included 3.4 and 3.5 into the travis build, along with a few minor fixes it seems to work!